app/lib/admin/backend.dart

```diff
@@ -271,6 +277,117 @@ class AdminBackend {
     });
   }
 
+  Future<void> removePackage(String packageName) async {
```

Comment:
Would it be possible to update the `remove_package.dart` script to use this method?

Reply:
Once we have this, let's remove the `remove_package.dart` script.